### ISSUE TITLE
[Requst] Import From KurobaEX

### ISSUE THREAD

**Issue by hbednar** [Adamantcheese/Kuroba#1439]
[KurobaEX](https://github.com/K1rakishou/Kuroba-Experimental) allows importing from Kuroba. 
It would be great if it worked the other way and users could also migrate from KurobaEX to Kuroba.

**Comment by Adamantcheese** [comment]
It would be best if 4chanX/Ex/this one all had a common import/export format instead of having bespoke importer/exporters for each application.

**Comment by hbednar** [comment]
> It would be best if 4chanX/Ex/this one all had a common import/export format instead of having bespoke importer/exporters for each application.

Yes that would be the best option but, KurobaEX is developed less often.

Also, KurobaEX can already import from Kuroba so it may not be that different.
